### DISCUSSION TITLE
sudo service netdata status returns "PROCFILE: Cannot open file '/proc/316237/status'" and a few other errors. What should I do?

### DISCUSSION THREAD

**Discussion by Danrancan** [netdata/netdata#15865] [Q&A]
I am running Ubuntu Server 20.04 on Raspberry Pi 4. It is a LEMP server. I have installed netdata with the installation script. When I run the command `sudo service netdata status` it returns the following:

```
Aug 21 20:30:57 mail.example.com apps.plugin[2799]: started on pid 2799
Aug 21 20:30:57 mail.example.com apps.plugin[2799]: set name of thread 2830 to APPS_READER
Aug 21 20:30:57 mail.example.com ebpf.plugin[2781]: PROCFILE: Cannot open file '/proc/316237/status'
Aug 21 20:30:57 mail.example.com ebpf.plugin[2781]: Cannot open /proc/316237/status
Aug 21 20:30:58 mail.example.com apps.plugin[2799]: Using now_boottime_usec() for uptime (dt is 2 ms)
Aug 21 20:30:58 mail.example.com debugfs.plugin[2789]: Zswap is disabled
Aug 21 20:30:58 mail.example.com debugfs.plugin[2789]: Failed to find powercap zones.
Aug 21 20:32:13 mail.example.com apps.plugin[2799]: Cannot process /proc/5240/io (command 'sh')
Aug 21 20:34:05 mail.example.com apps.plugin[2799]: Cannot process /proc/5820/cmdline (command 'sh')
Aug 21 21:06:39 mail.example.com apps.plugin[2799]: Cannot process /proc/14940/status (command 'grep')
```
What do these "Cannot process" errors mean and how can I fix them? Any help is much appreciated! Thanks!

**Comment by MrZammler** [comment]
Hi @Danrancan 

The plugin is trying to read some standard proc entries from these pids (it goes through all the pids running on the system). Likely it can't find them for some reason (perhaps the pids exited). It's not a fatal error, i.e. one that would cause issues.

**Comment by Danrancan** [comment]
> Hi @Danrancan 
> 
> 
> 
> The plugin is trying to read some standard proc entries from these pids (it goes through all the pids running on the system). Likely it can't find them for some reason (perhaps the pids exited). It's not a fatal error, i.e. one that would cause issues.

Okay thanks for letting me know that, but is there anything i can do about this error? I would prefer wiping my server clean of errors if possible.
